dev-tools/generate_documentation.sh

```diff
@@ -73,6 +73,7 @@ find ${SPHINX_DIR}/${SPHINX_APIDOC_EXT_DIR} -type f -name "*.rst" -print0 | xarg
 
 # Remove inherited members from tests
 find ${SPHINX_DIR}/${SPHINX_APIDOC_EXT_DIR} -type f -name "integreat_cms.cms.tests*.rst" -print0 | xargs -0 --no-run-if-empty sed --in-place '/:inherited-members:/d'
+find ${SPHINX_DIR}/${SPHINX_APIDOC_EXT_DIR} -type f -name "integreat_cms.api.tests*.rst" -print0 | xargs -0 --no-run-if-empty sed --in-place '/:inherited-members:/d'
```

Comment:
```suggestion
find ${SPHINX_DIR}/${SPHINX_APIDOC_EXT_DIR} -type f -name "integreat_cms.*.tests.*rst" -print0 | xargs -0 --no-run-if-empty sed --in-place '/:inherited-members:/d'
```
I think one wildcard matching for all apps is fine here.